earth_enterprise/docs/landing_page/manual/answer/6014171.html

```diff
@@ -17,79 +17,22 @@
 <a name="top_of_file"></a>
 <p><img src="../art/common/logo_col_126x48.png" width="126" height="48" alt="Google logo" /></p>
 <h1><a href="../answer/index.html">Google Earth Enterprise Documentation Home</a> | Developers</h1>
-<h2>Add Python Libraries to GEE Python</h2><p>Python v2.7.5 is bundled with GEE, providing functionality such as publishing and custom search plug-ins.</p>
+<h2>Add Python Libraries to GEE Python</h2>
+<p>GEE uses system Python with some extra libraries to support functionality such as publishing and custom search plug-ins.</p>
 
-<p>The standard Python v.2.7.5 libraries are included with GEE. Additional modules specific to GEE are installed in:</p>
+<p>Additional modules specific to GEE are installed in:</p>
 
-<p><code>/opt/google/gepython/Python-2.7.5/lib/python2.7/site-packages</code></p>
+<p><code>/opt/google/gepython/Python-2.7.x/lib/python2.7/site-packages</code> (where x is determined by the system Python version.</p>
 
 <ul>
   <li><code>psycopg2</code> is a PostgreSQL database adapter</li>
   <li><code>google/protobuf</code> is Google's data storage mechanism</li>
```

Comment:
this suggested change is not a breaking change also making separate ticket is not worth the time spent in issue tracking IMO, usually if its safe change not requiring much work ( like nits) it could  be done in same commit.